docs/4.3/user-manual.md

```diff
@@ -439,6 +439,10 @@ can join you through her terminal by typing:
 $ tsh join <session_ID>
 ```
 
+!!! note
+
+Joining sessions is not supported in recording proxy mode (where `session_recording` is set to `proxy`).
```

Comment:
Don't these also need to be indented? Regardless, we should try to keep to one style for these.

Reply:
You're right, good catch. Fixed.